clouddriver-ecs/src/main/java/com/netflix/spinnakerclouddriver/aws/ecs/EcsCloudProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnakerclouddriver.aws.ecs;
+
+import org.springframework.stereotype.Component;
+import com.netflix.spinnaker.clouddriver.core.CloudProvider;
+
+import java.lang.annotation.Annotation;
+
+//@Component
```

Comment:
Commenting this out prevents ECS to appear as a cloud provider at this time - is this sensical?  I can see arguments for both options, but would rather go for the same approach as you've used in the past while implementing new cloud providers @tomaslin @robzienert